tests/Transport/BaseTestCase.php

```diff
@@ -250,10 +308,127 @@ public function testIncorrectDataTypeExceptionPOST($data) {
 	public function dataIncorrectDataTypeException() {
 		return [
 			'boolean' => [true],
+			'integer' => [12345, '12345'],
+			'float'   => [12.345, '12.345'],
 			'object'  => [new stdClass()],
 		];
 	}
 
+	/**
+	 * Tests receiving an exception when the request() method received an invalid input type as `$options`.
+	 *
+	 * @dataProvider dataInvalidTypeNotArray
+	 *
+	 * @covers \WpOrg\Requests\Transport\Curl::request
+	 * @covers \WpOrg\Requests\Transport\Fsockopen::request
+	 *
+	 * @param mixed $input Invalid parameter input.
+	 *
+	 * @return void
+	 */
+	public function testRequestInvalidOptions($input) {
+		$this->expectException(InvalidArgument::class);
+		$this->expectExceptionMessage('Argument #4 ($options) must be of type array');
+
+		$transport = new $this->transport();
+		$transport->request('/', [], [], $input);
+	}
+
+	/**
+	 * Tests receiving an exception when the request_multiple() method received an invalid input type as `$requests`.
+	 *
+	 * @dataProvider dataRequestMultipleReturnsEmptyArrayWhenRequestsIsEmpty
+	 *
+	 * @covers \WpOrg\Requests\Transport\Curl::request
+	 * @covers \WpOrg\Requests\Transport\Fsockopen::request
+	 *
+	 * @param mixed $input Invalid parameter input.
+	 *
+	 * @return void
+	 */
+	public function testRequestMultipleReturnsEmptyArrayWhenRequestsIsEmpty($input) {
+		$transport = new $this->transport();
+		$this->assertSame([], $transport->request_multiple($input, $this->getOptions()));
+	}
+
+	/**
+	 * Data Provider.
+	 *
+	 * @return array
+	 */
+	public function dataRequestMultipleReturnsEmptyArrayWhenRequestsIsEmpty() {
+		return [
+			'null'        => [null],
+			'empty array' => [[]],
+		];
+	}
+
+	/**
+	 * Tests receiving an exception when the request_multiple() method received an invalid input type as `$requests`.
+	 *
+	 * @dataProvider dataRequestMultipleInvalidRequests
+	 *
+	 * @covers \WpOrg\Requests\Transport\Curl::request
+	 * @covers \WpOrg\Requests\Transport\Fsockopen::request
```

Comment:
```suggestion
	 * @covers \WpOrg\Requests\Transport\Curl::request_multiple
	 * @covers \WpOrg\Requests\Transport\Fsockopen::request_multiple
```

Reply:
Thanks for catching that! I completely missed, but this should also explain the CodeCov flag on this PR. Fixed now.